test/QuadForm/Herm/Genus.jl

```diff
@@ -395,4 +395,38 @@
+  
+  L = hermitian_lattice(E, generators = gene, gram_ambient_space = D)
+  gens, def, P0 = @inferred Hecke.genus_generators(L)
+  @test length(gens) == 0
+  @test !def
+  @test P0 == 1*base_ring(L)
```

Comment:
This is wierd, your documentation claims that `P0` is a prime ideal, but `1` is not a prime ideal.

Reply:
ah okay the reason is that in the indefinite case there is no need to run iterated neighbors.